README.rst

```diff
@@ -1,60 +1,55 @@
 Devstack
-#################
+########
 
-|ci-provisioning-badge| |ci-cli-badge| |doc-badge| |license-badge|
+|ci-provisioning-badge| |ci-cli-badge| |license-badge|
 |status-badge|
 
 
-DEPRECATION NOTICE
-******************
+Overview
+********
 
-Going forward, devstack will be primarily used for development by 2U. To do development
-on Open edX, it is recommended that `Tutor`_ be used instead.
+This is 2U's version of devstack used for developing using the Open edX platform.
 
-For more information on this deprecation, please visit the `associated deprecation ticket`_.
+The Open edX version of devstack was deprecated and archived. Be careful to
+avoid any outdated documentation from the Open edX Read the Docs or the
+openedx-unsupported GitHub org. For more information on the Open edX
+deprecation, please visit the `associated deprecation ticket`_.
 
-.. _Tutor: https://docs.tutor.edly.io/
 .. _associated deprecation ticket: https://github.com/openedx/devstack/issues/907
 
 
 Getting Started
 ***************
 
-The `Getting Started guide`_ lives with the rest of the documentation in Read the Docs.
+The `Getting Started guide`_ lives with the rest of the documentation.
 
-.. _Getting Started guide: https://edx.readthedocs.io/projects/open-edx-devstack/en/latest/getting_started.html
+.. _Getting Started guide: https://github.com/edx/devstack/blob/master/docs/getting_started.rst
```

Comment:
What's the reason for switching these URLs from the rendered docs to the source? Do we no longer have the ability to push to edx.readthedocs.io?

Reply:
Here is the ticket for publishing to Read the Docs again: https://github.com/edx/edx-arch-experiments/issues/578

Reply:
I see, thanks!